x-pack/plugins/profiling/public/components/primary_and_comparison_search_bar.tsx

```diff
@@ -67,7 +67,10 @@ export function PrimaryAndComparisonSearchBar() {
         <EuiTitle size="xxs">
           <h3>
             {i18n.translate('xpack.profiling.comparisonSearch.baselineTitle', {
-              defaultMessage: 'Baseline flamegraph',
+              defaultMessage:
```

Comment:
You'll have to use a different message entirely. These messages are translated once and use the defaultMessage as a fallback, the translated message doesn't allow for this conditional.

Reply:
Amended, please take another look.